resources/patches/ruby_241/aix_ruby_2.1_fix_proctitle.patch

```diff
@@ -0,0 +1,30 @@
+--- ruby-1.9.3-p547/missing/setproctitle.c	2013-02-05 21:13:40.000000000 -0800
++++ ruby-1.9.3-p547-patched/missing/setproctitle.c	2014-10-08 09:58:12.000000000 -0700
```

Comment:
Oh, because we don't apply it. The resource has been carried forward to 2.3 and now 2.4, but the line to apply the patch has been deleted. I'll remove this file.